src/caselawclient/xquery/set_metadata_this_uri.xqy

```diff
@@ -0,0 +1,15 @@
+xquery version "1.0-ml";
+
+declare namespace akn = "http://docs.oasis-open.org/legaldocml/ns/akn/3.0";
+declare variable $uri as xs:string external;
+declare variable $content as xs:string external;
+
+(xdmp:node-replace(
+    document($uri)/akn:akomaNtoso/akn:judgment/akn:meta/akn:identification/akn:FRBRWork/akn:FRBRuri,
```

Comment:
Do the Manifestation and Expression need to match, as well as the Work? (At this time, I think so.)

Somewhat concerned that this is quite a bit of editing in a different place -- is there someway we can make the Transformation Engine responsible for all this?